### ISSUE TITLE
Some  Questions On Your Code

### ISSUE THREAD

**Issue by bessics** [Zhuo-Cao/FlashVTG#11]
Thanks for your great work.I just have some doubts after reading Paper and Running code:
1. what do "lw_reg"  "lw_cls",  "lw_sal", "lw_saliency" mean in your code . I know  these are the weights of the loss functions mentioned in your paper, but which part of the loss does each of them correspond to?  

![Image](https://github.com/user-attachments/assets/17445873-79de-4632-ae33-8d775c54fa68)

2. As you mentioned in your paper, only the **SampledNCE Loss and Saliency Loss** were used in the HD task. However, in FlashVTG/scripts/tvsum/train.sh and the opt.json  you provided, none of the  weights for each loss function are set to zero.
3. I find the configuration in the red box of the following figure doesn't appear in the config.py , nor does it appear in the train.sh .What should I do to use these configurations
<img width="270" alt="Image" src="https://github.com/user-attachments/assets/99e61e33-65c2-4d2d-ac53-cfa9f59a419e" />

Thanks for your reply

**Comment by Zhuo-Cao** [comment]
1. Please refer to `data/MR.py` for the relevant implementation details.
2. During training, you may choose which loss functions to enable in `FlashVTG/model.py`. Please note that any loss weights specified in the configuration will only take effect if their corresponding loss functions are activated.
3. Feel free to disregard them.

Thanks 
Zhuo

**Comment by bessics** [comment]
Thank you for your detailed explanation - I've gone through the code and everything makes perfect sense now.
